### PR TITLE
Update FeatureGate when Bump Release

### PR DESCRIPTION
**Pull request by wzshiming**

### CODE REVIEW

File: pkg/kwokctl/k8s/feature_gates.go [hunk 1]
```diff
@@ -23,7 +23,8 @@ import (
 	"strings"
 )
 
-//go:generate ./feature_gates_data.sh 24
+// Move to another to execute
+////go:generate ./feature_gates_data.sh
```
Comment (yibozhuang) [inline pkg/kwokctl/k8s/feature_gates.go:27]: remove this if its not used?

File: pkg/kwokctl/k8s/feature_gates_data.sh [hunk 2]
```diff
@@ -16,20 +16,32 @@
...
 
-latest_release="${1:-24}"
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+latest_release="${1:-}"
+minimum_release=6
```
Comment (yibozhuang) [inline pkg/kwokctl/k8s/feature_gates_data.sh:26]: can you add a comment on why minimum_release is 6 here?

File: pkg/kwokctl/k8s/feature_gates_data.sh [hunk 3]
```diff
@@ -16,20 +16,32 @@
...
 function features() {
-    for i in $(seq 6 "${latest_release}"); do
-        curl -sSL "https://github.com/kubernetes/kubernetes/raw/release-1.${i}/pkg/features/kube_features.go" |
-            grep "{Default: " |
-            sed -e 's/\w\+\.//g' |
-            sed -e 's/[:,}]//g' |
-            awk "{print \$1, \$5, $i}"
-    done
+  for i in $(seq "${minimum_release}" "${latest_release}"); do
+    curl -sSL "https://github.com/kubernetes/kubernetes/raw/release-1.${i}/pkg/features/kube_features.go" |
```
Comment (yibozhuang) [inline pkg/kwokctl/k8s/feature_gates_data.sh:35]: nit: maybe we should consider not hard coding 1 here but not required.

the minimum_release to latest_release should include the 1.

Comment (wzshiming) [reply pkg/kwokctl/k8s/feature_gates_data.sh:37]: Kubernetes added Feature Gate in 1.6

File: pkg/kwokctl/k8s/feature_gates_data.sh (hunk 2 above)
Comment (wzshiming) [reply pkg/kwokctl/k8s/feature_gates_data.sh:28]: Kubernetes added Feature Gate in 1.6

File: pkg/kwokctl/k8s/feature_gates_data.sh (hunk 3 above)
Comment (wzshiming) [reply pkg/kwokctl/k8s/feature_gates_data.sh:37]: It's not easy to calculate Semver

File: pkg/kwokctl/k8s/feature_gates_data.sh (hunk 2 above)
Comment (wzshiming) [reply pkg/kwokctl/k8s/feature_gates_data.sh:28]: Updated

**Review by yibozhuang — APPROVED** [review]
/lgtm

**Pull request merged**